docs/using-the-compiler.rst

```diff
@@ -287,8 +287,14 @@ Input Description
               // Improve allocation of stack slots for variables, can free up stack slots early.
               // Activated by default if the Yul optimizer is activated.
               "stackAllocation": true,
-              // Select optimization steps to be applied.
-              // Optional, the optimizer will use the default sequence if omitted.
+              // Select optimization steps to be applied. It is also possible to modify both the
+              // optimization sequence and the cleanup sequence. Instructions for each sequence
+              // are separated with the : delimiter. For more information see
+              // "The Optimizer > Selecting Optimizations".
+              // This field is optional and if not provided, the default sequences for both
+              // optimization and cleanup are used.
+              // If set to an empty value, only the default cleanup sequence is used and
```

Comment:
This is a bit misleading, and ambiguous as well. If you set either the optimizer sequence or the clean-up sequence as an empty value, then no optimizer and/or cleanup sequence will be applied.
E.g. if you set the steps as `:dhfG`, no optimizer steps will be executed, whereas the clean-up sequence will be executed as provided, i.e. `dhfG`. Neither of these is default. If you however set an empty non delimited sequence, in the case the default optimizer and clean-up sequence will be executed.

Also, please change each instance of `cleanup` to `clean-up`.

Reply:
I have changed all the `cleanup` to `clean-up` and clarified the language as requested. Let me know what you think.